File: partner-portal/src/API/ViewModels/Driver.cs
using Rsbc.Dmf.IcbcAdapter;

namespace Rsbc.Dmf.PartnerPortal.Api.ViewModels
{
    public class Driver
    {
        public string Id { get; set; }
        public bool? Flag51 { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string DriverLicenseNumber{ get; set; }
        public bool? LoadedFromICBC { get; set; }
        public DateTimeOffset? MedicalIssueDate { get; set; }
        public DateTimeOffset? BirthDate { get; set; }
        public string Sex { get; set; }
        public string AddressLine1 { get; set; }
        public string City { get; set; }
        public string Province { get; set; }
        public string Postal { get; set; }
        public string Country { get; set; }
        public string LicenceClass { get; set; }
        public string MasterStatusCode { get; set; }
        public DateTimeOffset? LicenceExpiryDate { get; set; }
        public List<string> RestrictionCodes { get; set; }
        public string Height { get; set; }
        public string Weight { get; set; }
        public string SecurityKeyword { get; set; }
        public List<DriverStatus> Status { get; set; }
        public List<DriverMedicals> Medicals { get; set; }
    }
}

File: partner-portal/src/UI/src/app/shared/api/models/driver.ts
/* tslint:disable */
/* eslint-disable */
import { DriverMedicals } from '../models/driver-medicals';
import { DriverStatus } from '../models/driver-status';
export interface Driver {
  addressLine1?: string | null;
  birthDate?: string | null;
  city?: string | null;
  country?: string | null;
  driverLicenseNumber?: string | null;
  firstName?: string | null;
  flag51?: boolean | null;
  height?: string | null;
  id?: string | null;
  lastName?: string | null;
  licenceClass?: string | null;
  licenceExpiryDate?: string | null;
  loadedFromICBC?: boolean | null;
  masterStatusCode?: string | null;
  medicalIssueDate?: string | null;
  medicals?: Array<DriverMedicals> | null;
  postal?: string | null;
  province?: string | null;
  securityKeyword?: string | null;
  sex?: string | null;
  status?: Array<DriverStatus> | null;
  weight?: string | null;
}
